test/createNewProject.test.ts

```diff
@@ -20,19 +20,17 @@ import { validateVSCodeProjectFiles } from './initProjectForVSCode.test';
 // tslint:disable-next-line:no-function-expression max-func-body-length
 suite('Create New Project Tests', async function (this: ISuiteCallbackContext): Promise<void> {
     this.timeout(60 * 1000);
-
     const testFolderPath: string = path.join(os.tmpdir(), `azFunc.createNewProjectTests${fsUtil.getRandomHexString()}`);
-    const outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel('Azure Functions Test');
-    ext.outputChannel = outputChannel;
 
     // tslint:disable-next-line:no-function-expression
     suiteSetup(async function (this: IHookCallbackContext): Promise<void> {
         this.timeout(120 * 1000);
+        const outputChannel: vscode.OutputChannel = vscode.window.createOutputChannel('Azure Functions Test');
```

Comment:
Honestly you probably don't need to create and dispose of outputChannels in these tests anymore. You can just pass in `ext.OutputChannel` if necessary (it should already be initialized since you made sure the extension is activated as a part of this PR). That might fix the failing tests in the CI, too